job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/Constants.groovy

```diff
@@ -23,6 +23,10 @@ class Constants {
     static final String BRANCH = "master"
     static final String GITHUB_ORG_UNIT = "kiegroup"
     static final String PULL_REQUEST_FOLDER = "pullrequest"
+    static final String FDB_FOLDER = "fullDownstream"
```

Comment:
`downstream` would probably be a more appropriated name

Reply:
actually is that correct as downstream term? I think not as FDB is actually FULL BUILD from first to end of repository list.
So correctly this is even not downstream, but just FULL BUILD
Downstream would be from PR repo down only, but we do also upstream repo build